doc/source/whatsnew/v0.24.0.txt

```diff
@@ -29,6 +29,8 @@ Datetimelike API Changes
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 - For :class:`DatetimeIndex` and :class:`TimedeltaIndex` with non-``None`` ``freq`` attribute, addition or subtraction of integer-dtyped array or ``Index`` will return an object of the same class (:issue:`19959`)
+- Subtraction of :class:`Period` from :class:`Period` will return a :class:`DateOffset` object instead of an integer (:issue:`21314`)
```

Comment:
can you make a new sub-section and show a previous / new behavior example.

Reply:
The "can" part of that is a very good question.  I'll try.

Reply:
your subsection looks good, you don't need this note here any longer